.gitmodules

```diff
@@ -23,3 +23,6 @@
 	path = ext/ffmpeg
 	url = https://github.com/FFmpeg/FFmpeg.git
 	shallow = true
+[submodule "ext/libavif"]
+	path = ext/libavif
+	url = https://github.com/AOMediaCodec/libavif.git
```

Comment:
This is the result of the following command:

```
git submodule add https://github.com/AOMediaCodec/libavif.git ext/libavif
```

Reply:
Done at https://github.com/sandflow/libench/pull/18/commits/54f3cf426701cb2af6676be9419c3b9786ee35b0